src/main/webapp/app/exercises/programming/shared/code-editor/actions/code-editor-resolve-conflict-modal.component.ts

```diff
@@ -13,16 +13,16 @@ import { GitConflictState } from 'app/exercises/programming/shared/code-editor/m
 export class CodeEditorResolveConflictModalComponent {
     constructor(public activeModal: NgbActiveModal, private repositoryService: CodeEditorRepositoryService, private conflictService: CodeEditorConflictStateService) {}
```

Comment:
CodeEditorRepositoryService and CodeEditorConflictStateService can be removed, the provided service as well